### ISSUE TITLE
How can cronjobs made persistent during a reboot with UEFI secure boot enabled

### ISSUE THREAD

**Issue by DVTrast** [lamw/ghettoVCB#303]
Hello.
Do make the backup automatically, it make sense to create a cronjob for it (as described). But with ESX8 and UEFI secure boot enabled, I´m unable to make the cronjob (in /var/spool/cron/crontabs/root) persistent - is there a was to do it?

In the file "/etc/rc.local.d/local.sh" the following is present:
`Note: This script will not be run when UEFI secure boot is enabled.`

Thank you

Best regards

Stefan Rapp

**Comment by lamw** [comment]
No. This is the expected behavior if you've got Secure Boot enabled (see https://williamlam.com/2018/06/using-esxi-kickstart-firstboot-with-secure-boot.html for more details)

Your options are either disable Secure Boot (not ideal) or schedule the invocation remotely rather than directly on the ESXi host

**Comment by DVTrast** [comment]
@lamw Ok - Thank you for your answer :-)

**Comment by Giga-Pudding** [comment]
Hi,

i don't know if ESXi uses Systemd, but if yes, you could use Systemd timers.

BR,
Giga-Pudding

**Comment by DVTrast** [comment]
@Giga-Pudding thank´s
I´ve build an workaround and trigger the script from an other server via ssh.
